client/components/Settings.js

```diff
@@ -15,20 +15,48 @@ import CustomBankField from './CustomBankField';
 import {OpCatChartTypeSelect, OpCatChartPeriodSelect} from './Charts';
 import T from './Translated';
 
-
 class Account extends React.Component {
 
+    constructor(props) {
+        super(props);
+        this.state = {
+            isDefault: (store.getDefaultAccountId() === this.props.account.id)
+        };
+    }
+
+    componentDidMount() {
+        store.on(State.defaultAccount, function() {
+            this.setState({isDefault: (store.getDefaultAccountId() === this.props.account.id)});
+        }.bind(this));
+    }
+
     onDelete(id) {
         Actions.DeleteAccount(this.props.account);
     }
 
-    render() {
-        var a = this.props.account;
+    setAsDefault(id) {
+        Actions.SetDefaultAccount(this.props.account);
+    }
 
-        var label = a.iban ? `${a.title} (IBAN: ${a.iban})`
-                           : a.title;
+    render() {
+        let a = this.props.account;
+        let label = a.iban ? `${a.title} (IBAN: ${a.iban})` : a.title;
+        let defaultAccountTitle = t("settings.set_default_account") || "Set as default account";
+        let selected = "";
+
+        if (this.state.isDefault) {
+            defaultAccountTitle = "";
+            selected = " selected";
+        }
 
-        return <tr>
+        return <tr className="bankAccount">
+            <td>
+                <span className={"glyphicon glyphicon-star clickable" + selected}
```

Comment:
Instead of `glyphicon`, can you use FontAwesome? There's `star-o` (empty star) and `star` (full star) for that, and it will fit in perfectly with the rest of the UI :)